### DISCUSSION TITLE
Launch of the 2024 Web Almanac & Call for Contributions

### DISCUSSION THREAD

**Discussion by nrllh** [HTTPArchive/almanac.httparchive.org#3619] [General]
Dear all,

We are excited to kick off the 2024 Web Almanac and invite contributions to this year's edition! This year together with @cqueern, we are going to lead this project. If you want to know more about the changes this year, please read [<ins>this section</ins>](#whats-new-this-year).

<a id="contribute-to-the-2024-web-almanac"></a>
## [Contribute to the 2024 Web Almanac](#contribute-to-the-2024-web-almanac)

You can contribute to this year’s version as an [author](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Authors'-Guide), [reviewer](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Reviewers'-Guide),  [analyst](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Analysts'-Guide), or [editors](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Editors'-Guide). If you are interested in contributing, please visit the provided link for the chapters  and indicate which role you are interested in. Alternatively, you can nominate yourself or someone else using [this nomination form](https://forms.gle/XQaHTw3Gq6Fki169A).

**Page Content**
* [CSS](https://github.com/HTTPArchive/almanac.httparchive.org/issues/3591)
* [JavaScript](https://github.com/HTTPArchive/almanac.httparchive.org/issues/3592)
* [Markup](https://github.com/HTTPArchive/almanac.httparchive.org/issues/3593)
* [Structured Data](https://github.com/HTTPArchive/almanac.httparchive.org/issues/3594)
* [Fonts](https://github.com/HTTPArchive/almanac.httparchive.org/issues/3595)
* [Media](https://github.com/HTTPArchive/almanac.httparchive.org/issues/3596)
* [WebAssembly](https://github.com/HTTPArchive/almanac.httparchive.org/issues/3597)
* [Third Parties](https://github.com/HTTPArchive/almanac.httparchive.org/issues/3598)
* [Interoperability](https://github.com/HTTPArchive/almanac.httparchive.org/issues/3599)
* [Cookies 🆕](https://github.com/HTTPArchive/almanac.httparchive.org/issues/3617)

**User Experience**
* [SEO](https://github.com/HTTPArchive/almanac.httparchive.org/issues/3600)
* [Accessibility](https://github.com/HTTPArchive/almanac.httparchive.org/issues/3601)
* [Performance](https://github.com/HTTPArchive/almanac.httparchive.org/issues/3602)
* [Privacy](https://github.com/HTTPArchive/almanac.httparchive.org/issues/3603)
* [Security](https://github.com/HTTPArchive/almanac.httparchive.org/issues/3604)
* [Mobile Web](https://github.com/HTTPArchive/almanac.httparchive.org/issues/3605)
* [Capabilities](https://github.com/HTTPArchive/almanac.httparchive.org/issues/3606)
* [PWA](https://github.com/HTTPArchive/almanac.httparchive.org/issues/3607)

**Content Publishing**
* [CMS](https://github.com/HTTPArchive/almanac.httparchive.org/issues/3608)
* [Ecommerce](https://github.com/HTTPArchive/almanac.httparchive.org/issues/3609)
* [Jamstack](https://github.com/HTTPArchive/almanac.httparchive.org/issues/3610)
* [Sustainability](https://github.com/HTTPArchive/almanac.httparchive.org/issues/3611)

**Content Distribution**
* [Page Weight](https://github.com/HTTPArchive/almanac.httparchive.org/issues/3612)
* [CDN](https://github.com/HTTPArchive/almanac.httparchive.org/issues/3613)
* [Compression](https://github.com/HTTPArchive/almanac.httparchive.org/issues/3614)
* [Caching](https://github.com/HTTPArchive/almanac.httparchive.org/issues/3615)
* [HTTP](https://github.com/HTTPArchive/almanac.httparchive.org/issues/3616)

**Non-technical** 
* [Trends🆕](https://github.com/HTTPArchive/almanac.httparchive.org/issues/3618)

Feel free to propose new chapters in the comments that you believe would complement the 2024 Web Almanac.

<a id="whats-new-this-year"></a>
## [What’s new this year?](#whats-new-this-year)

The project has been greatly maintained by the HTTPArchive’s core team until now and we are thankful for their great effort. However, managing the Web Almanac project demands a considerable amount of time. Thus, it is not sustainable to have only a few people overseeing such an extensive project. That's why we are transitioning the project management of the Web Almanac to be more community-driven, with distributed responsibilities.

The progression of the project remains consistent with previous years, which means we will continue employing the same strategies and tools. The primary changes are focused on project management.

### Organizing Committee

We are excited to share the initial structure of this year's organizing committee. The committee is still being formed, and we will provide a complete list of members soon. 

| Role                	| Description                              	| Chair |
|-------------------------|----------------------------------------------|-------|
| General Chairs       	| Overall project leadership               	| Nurullah Demir (@nrllh), Caleb Queern (@cqueern)  |
| Program Committee Chairs | Manages content and chapter milestones   	| Tobias Urban (@turban1988)    |
| Chapter Chairs       	| Leads chapter; main content author       	| _listed in the issues_  |
| Publication Chair   	| Manages publishing processes            	| Rick Viscomi (@rviscomi), Barry Pollard (@tunetheweb)  |
| Logistic Chair      	| Manages collaboration tools              	| Christian Böttger (@ChrisBeeti) |
| Development Chair | Handles design and development tasks   | Saptak Sengupta (@SaptakS) |
| Media Chair         	| Manages news and media relations         	| Miriam Naß |
| Prize Chair         	| Coordinates awards distribution          	| Günes Acar (@gunesacar) |
| Event Chair         	| Organizes virtual conference logistics   	| Henri Helvetica (@henrihelvetica)   |

### Virtual Conference

Upon the completion of this year's edition, we plan to organize a virtual conference where contributors can present their findings.

### Awards

This year, awards will be given to the best chapter(s). The Prize Chair will oversee this process.  Stay tuned for more details on the award process.

### Non-technical Chapter

We believe that there is huge potential for content analysis using HTTPArchive data. To delve into this domain, we are introducing a non-technical section. Our goal is to uncover insights into various trends — social, economic, political, and technical — reflected in the webpage content we crawl and evaluate. Methodologies like language models (e.g., BERT) offer promising opportunities for such analysis. More details can be found on the chapter's page, inviting contributions to explore these.

<a id="deadlines"></a>
## [Deadlines](#deadlines)

We have set the following deadlines:

| # | Milestone                                  	| Deadline   |
|----|------------------------------------------------|------------|
| 1  | Call for contributions   	| 03/01/2024 |
| 2  | Complete program and content committee     	| 04/15/2024 |
| 3  | First meeting to outline the chapter contents  | 05/01/2024 |
| 4  | Custom metrics completed                   	| 06/01/2024 |
| 5  | HTTP Archive crawl                                  	| 06/01/2024 |
| 6  | Querying all metrics and saving the results	| 08/15/2024 |
| 7  | First draft of chapter                     	| 09/15/2024 |
| 8  | Reviewing & Editing of chapter             	| 10/01/2024 |
| 9  | Publication of chapter (Markdown & PR)     	| 10/15/2024 |
| 10 | Launch of the 2024 Web Almanac              	| ~~11/01/2024~~ 11/11/2024  |
| 11 | [Virtual Conference](https://www.youtube.com/watch?v=cdYR0ZmplIM)                         	|  ~~11/20/2024~~ 12/10/2024  |

We kindly invite @HTTPArchive/almanac-contributors to the success of this year's edition!

We look forward to your contributions and are excited to collaborate on another successful edition of the Web Almanac!

**Comment by airbr** [comment]
> Our goal is to uncover insights into various trends — social, economic, political, and technical — reflected in the webpage content we crawl and evaluate.

I am seriously interested in this!

Also, a side note, I would love to see some  info or relateable content about what the community driven aspect will look like. Infographics or pictures about previous editions and contributors would give me an idea what the community was like.

**Reply by airbr** [reply]
![Screenshot 2024-03-09 at 5 15 15 PM](https://github.com/HTTPArchive/almanac.httparchive.org/assets/18056682/1f329a85-01c0-4986-8842-a8842afd9851)

This is good ! more of this! gives a human face to the project :D

**Reply by nrllh** [reply]
Thank you! Please feel free to comment the issue, so we know who wants to contribute to the chapter.
